WordPress/Sniffs/DB/PreparedSQLPlaceholdersSniff.php

```diff
@@ -23,6 +23,14 @@
  *   placeholders found or individual parameters for each placeholder should
  *   be passed.
  *
+ * The sniff allows for a specific pattern with a variable number of placeholders
+ * created using code along the lines of:
+ * `sprintf( 'query .... IN (%s) ...', implode( ',', array_fill( 0, count( $something), '%s' ) ) )`.
```

Comment:
There is a space missing after `$something`.

Reply:
Good point. Fixed.

For the example I agree that the space should be there. As for the sniff: that should work independently of code-style.